Source/Graph/Graph.cs

```diff
@@ -586,6 +586,49 @@ public List<Node> SuccessorsAsList(Node n)
       }
     }
 
+    // this function gives a simpler way to compute dominators but it assmumes the graph is a DAG.
```

Comment:
"this function" -> "This method"

"in the topological order" -> "in topological order"

remove period after "order"

"The function" -> "This method"